manifests/dev/vsphere-7.0u2/guestcluster/1.17/pvcsi.yaml

```diff
@@ -411,3 +415,22 @@ metadata:
   name: internal-feature-states.csi.vsphere.vmware.com
   namespace: {{ .PVCSINamespace }}
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: vsphere-csi-controller
+  namespace: {{ .PVCSINamespace }}
+  labels:
+    app: vsphere-csi-controller
+spec:
+  ports:
+    - name: ctlr
+      port: 2112
+      targetPort: 2112
+      protocol: TCP
+    - name: syncer
+      port: 2113
+      targetPort: 2113
+      protocol: TCP
+  selector:
+    app: vsphere-csi-controller
```

Comment:
Can you confirm if this yaml can be applied on a tkg cluster out of the box. I see a "no newline at end of file" warning.